front/types/triggers/test_webhook_source_presets.ts

```diff
@@ -44,7 +45,9 @@ class TestWebhookService implements RemoteWebhookService {
       Error
     >
   > {
-    logger.info("Creating webhooks with params:", params);
+    logger.info(
+      `Creating webhooks for test preset in workspace ${params.auth.getNonNullableWorkspace().sId}`
+    );
```

Comment:
let's pass the workspace id in the properties to make filtering in dd more easy
```suggestion
    logger.info(
      { workspaceId: auth.getNonNullableWorkspace().sId },
      `Creating webhooks for test preset`
    );
```